api/turing/validation/validator_test.go

```diff
@@ -212,27 +213,29 @@ func (tt routerConfigTestCase) RouterConfig() *request.RouterConfig {
 		LogConfig: &request.LogConfig{
 			ResultLoggerType: "nop",
 		},
+		Ensembler: tt.ensembler,
 	}
 }
 
 func TestValidateTrafficRules(t *testing.T) {
+	// Common variables used by suite tests
+	routeAID, routeBID := "route-a", "route-b"
+	routeA, routeB := &models.Route{
```

Comment:
Minor clean up of the hardcoded route info in the tests, to use the variables here.